Description: adds a man page
 TODO: Put a short summary on the line above and replace this paragraph
 with a longer explanation of this change. Complete the meta-information
 with other relevant fields (see below for details). To make it easier, the
 information below has been extracted from the changelog. Adjust it or drop
 it.
 .
 invisicalc (0.1.0-1ubuntu2) UNRELEASED; urgency=medium
 .
   [ invisible ]
   * Initial release.
   *
 .
   [ Jordan Blakey ]
   *
Author: Jordan Blakey <jordan.t.blakey@gmail.com>

---
The information above should follow the Patch Tagging Guidelines, please
checkout https://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: (upstream|backport|vendor|other), (<patch-url>|commit:<commit-id>)
Bug: <upstream-bugtracker-url>
Bug-Debian: https://bugs.debian.org/<bugnumber>
Bug-Ubuntu: https://launchpad.net/bugs/<bugnumber>
Forwarded: (no|not-needed|<patch-forwarded-url>)
Applied-Upstream: <version>, (<commit-url>|commit:<commid-id>)
Reviewed-By: <name and email of someone who approved/reviewed the patch>
Last-Update: 2026-01-03

--- invisicalc-0.1.0.orig/meson.build
+++ invisicalc-0.1.0/meson.build
@@ -86,6 +86,12 @@ subdir('data')
 subdir('src')
 subdir('po')
 
+# Find man pages defined in the new man/ directory
+man_pages = ['man/invisicalc.1']
+
+# Install them to the standard system location
+install_man(man_pages)
+
 gnome.post_install(
      glib_compile_schemas: true,
     gtk_update_icon_cache: true,
